Packs/PANOStoCDLMonitoring/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,8 @@
+
+#### Scripts
+##### PANOStoCortexDataLakeMonitoring
+- Fixed an issue with the managing of the successful and failed Firewalls lists.
```

Comment:
the script was not modified, is that intensional?
if not, please remove

Reply:
No, I missed a revert in the commits. fixed